projects/packages/forms/src/blocks/shared/hooks/use-form-wrapper.js

```diff
@@ -6,29 +6,27 @@ import { __ } from '@wordpress/i18n';
 import { FORM_BLOCK_NAME } from '../util/constants';
 
 export default function useFormWrapper( { attributes, clientId, name } ) {
-	const BUTTON_BLOCK_NAME = 'jetpack/button';
-	const SUBMIT_BUTTON_ATTR = {
-		text: __( 'Submit', 'jetpack-forms' ),
-		element: 'button',
-		lock: { remove: true },
-	};
-
```

Comment:
I've moved these inline, as otherwise these variables are being created on every render of every field. They're only needed when performing the replaceBlock action